packages/components/src/config/types.ts

```diff
@@ -6,6 +6,7 @@ export const uiVariants = [
     'info',
     'warning',
     'destructive',
+    'disabled',
```

Comment:
I think we are maybe mixing too many visual states here 🤔

I would prefer to do it this way:

```
<Input variant="default" disabled />
```

Reply:
I've added it mainly because of Text component where I've run several times into instances where this is done via color. I think it's better than adding a disabled prop to Text?